### DISCUSSION TITLE
Vertical Space  for Feature Tracks

### DISCUSSION THREAD

**Discussion by jbrestel** [GMOD/jbrowse-components#3858] [Ideas]
Vertical scrolling per track is not obvious.  Often users won't realize data is not being shown.  Not sure the right answer but one suggestion is to scale to the available vertical space.  ie. always show all of the features but make them small enough to fit into the vertical space for the track.  (there is some track config to clip based on the feature depth?   seems like this could just as easily use the number of rows of features as the depth and then scale their size based on the track vertical space).  this would be more similar to what happens when you resize a Quant Track.  The entire plot/data is shown always but just gets resized.

**Comment by cmdcolin** [comment]
this is definitely an interesting idea and a long standing issue with per track scrolls, I have been thinking of ways to create more compact displays in general and I like that this would make doing so very interactive. I have done that interactive resizing also on 'arc display' from alignments track. for various reasons, it may be fairly involved to try to make this happen on plain 'feature tracks' like genes, variants, etc but definitely worth noting

more discussion related to this here https://github.com/GMOD/jbrowse-components/issues/534
